src/plugins/saved_objects_management/public/lib/resolve_import_errors.ts

```diff
@@ -27,13 +27,11 @@ interface RetryObject {
   replaceReferences?: any[];
 }
 
-async function callResolveImportErrorsApi(file: File, retries: any) {
+async function callResolveImportErrorsApi(http: HttpStart, file: File, retries: any) {
```

Comment:
Yea. Honestly the whole `src/plugins/saved_objects_management/public/lib` utilities are a mess, with honourable mentions for this file and `resolve_saved_objects`. I want to avoid as must as possible to modify them until I got a clearer vision on what needs to be done to get rid of the `SavedObjectLoader` usages (in https://github.com/elastic/kibana/issues/62048), as this will impact a lot of the `lib` files.